### ISSUE TITLE
Latest News "Home page"

### ISSUE THREAD

**Issue by cbruch** [Kducey1/BigSkyInternetDesign#9]
Can we just do 3 across?

**Comment by Kducey1** [comment]
Yes, it should be three across now!
Are there sites that you want me to put in the latest news?

**Comment by cbruch** [comment]
Let's Change the latest News to New or if you can come up with something cleaver.  What's Happen???

On the current BSID site is a big blue image that is the magazine article, I'd like that smaller & in the middle & linked to this page. By Melinda Davis

http://bigskyinternetdesign.com/images/ShowHorseTodayJune2016.pdf

**Comment by Kducey1** [comment]
Where on the home page would you like this?

**Comment by cbruch** [comment]
Humm that is a tough one.. I think as neat as the home page is now it would mess it up. What do you think?

**Comment by Kducey1** [comment]
I think it could potentially mess up the "clean look" that the page has going for it

**Comment by cbruch** [comment]
I agree..  Let's save the image & the pdf as well in case we want to use it.
